src/fulltrade/FullTradeCard.js

```diff
@@ -57,11 +57,11 @@ export default class BaseTradeCard extends Component {
                     assetName={selectedSymbol}
                     onClosePanel={() => actions.removeTrade(index)}
                 />
-                {tick && <TradeChart
+                {(tick && Object.keys(tick).length > 0) && <TradeChart
```

Comment:
Why is tick now an object instead of array?
I think we can make selector always return array, so that we dont have to check `undefined` here, just length will do, 

In addition, seems @borisyankov suggested to let chart handle empty array, so we could probably just pass it in, and enforce the props type so that it warns when it's undefined